app/instance-initializers/bugsnag.js

```diff
@@ -22,6 +23,12 @@ export default {
       Ember.onerror = function(error) {
         Bugsnag.context = getContext(router);
         const metaData = getMetaData(error, owner);
+        const user = getUser(owner);
+
+        if (user && typeof user === 'object') {
```

Comment:
I would suggest we change the call to have a default e.g. `const user = getUser(owner) || {};`